inference-engine/tests/functional/plugin/shared/include/single_layer_tests/binary_convolution.hpp

```diff
@@ -0,0 +1,48 @@
+// Copyright (C) 2019 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <memory>
+#include <string>
+#include <tuple>
+#include <vector>
+
+#include "functional_test_utils/layer_test_utils.hpp"
+#include "ngraph_functions/builders.hpp"
+#include "ngraph_functions/utils/ngraph_helpers.hpp"
+
+// ! [test_binary_convolution:definition]
```

Comment:
I just copied and modified them from `convolution.hpp`, and made them looks consistent. 
If it's needed, I will remove them.

Reply:
Oh interesting. I looked to see if it appears elsewhere, and found it in `docs/IE_PLUGIN_DG/PluginTesting.md`. I think it is a tag on the source code to help the documentation find it.

Since we do not want binary_convolution to be found from the documentation for a regular convolution, please remove it. (And do analogously for the other comment like this.)

Reply:
OK, got it!